src/sentry/seer/autofix/issue_summary.py

```diff
@@ -341,23 +279,19 @@ def _generate_summary(
     if not serialized_event or not event:
         return {"detail": "Could not find an event for the issue"}, 400
 
-    # get trace connected issues
-    connected_issues = _get_trace_connected_issues(event)
-
-    # get recommended event for each connected issue
-    serialized_events_for_connected_issues = []
-    filtered_connected_issues = []
-    for issue in connected_issues:
-        serialized_connected_event, _ = _get_event(issue, user)
-        if serialized_connected_event:
-            serialized_events_for_connected_issues.append(serialized_connected_event)
-            filtered_connected_issues.append(issue)
+    trace_tree = None
+    if event:
+        try:
+            trace_tree = _get_trace_tree_for_event(event, group.project, timeout=3)
```

Comment:
curious how long this call typically takes. if it depends on the size of the trace, do we need to limit depth, for example?